Update dependencies to secure versions

haqadn opened the pull request:
Fixes #27 

`ansi-regex`, `tmpl`, and `axios` are fixed by a simple update. `bi` was down the dependency chain started by `rollup-plugin-node-builtins`. While trying to fix that, I found that this dependency along with `rollup-plugin-node-globals` is no longer maintained. And the package called `rollup-plugin-polyfill-node` combines both and is maintained. It seems to be the recommendation by rollup as well:
![image](https://user-images.githubusercontent.com/3737780/148759243-563f3fbc-6e59-4712-8001-bc92458e7c0b.png)


However, I am not very confident about the change and @thingalon's thorough look is needed here.

haqadn committed:
Update npm dependencies and fix ansi-regex, tmpl, and axios

haqadn committed:
Replace node polyfill

Replace `rollup-plugin-node-builtins` and `rollup-plugin-node-globals` with  `rollup-plugin-node-globals` as the former dependencies are no longer maintained.

thingalon reviewed (changes requested):
I'm comfortable with the changes, but there is a linting issue that needs to be fixed:
https://github.com/Automattic/jetpack-boost-critical-css-gen/runs/4760561245?check_suite_focus=true

haqadn committed:
Fix lint

I have configured `requireConfigFile` to `false` as we apparently did not have any
babel config file.

haqadn commented:
I have configured `requireConfigFile` to `false` as we apparently did not have any
babel config file. Correct me if I am wrong @thingalon

thingalon reviewed (approved):
👍
